README.md

```diff
@@ -68,3 +68,7 @@ Generates this output (looks better in terminal):
    ▀▀▀▀▀▀▀ ▀▀  ▀   ▀  ▀    
                            
 ```
+
+## MSRV
+
+Current MSRV is 1.48.0
```

Comment:
optional nit: Might be worth expanding the acronyms (I assume that MSRV means "Minimal supported Rust vesion" / "Minimal supported version of the Rust compiler", right?).